puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/pipeline/katello-devel-forklift-test.yaml

```diff
@@ -4,6 +4,7 @@
     project-type: pipeline
     sandbox: true
     triggers:
+      - timed: "* */12 * * *"
```

Comment:
I think we may want this to read `0 */12 * * *` to be specific that we want it to run at minute 0 every 12 hours, rather than every minute

Reply:
@ehelms good catch! Not sure what happened there. 

I switched it to once a day `0 12 0 0 0`, since the nightly pipepline seems to run at night or the early morning. So this would get a run at night and then one guaranteed during the day  when devs are working